java/src/client/game/GameManager.java

```diff
@@ -60,22 +82,24 @@ public void stopPoller() {
      * When the poller finds out the server has a new version it will call this function to update the client game
      * @param cm the new clientModel
      */
-    public void updateClientModels(ClientModel cm){
+    public void updateGameManager(ClientModel cm){
```

Comment:
style choice. average person would not have trouble understanding. can change it for you if you need tho